server/sonar-web/src/main/js/apps/projectActivity/components/ProjectActivityAppContainer.js

```diff
@@ -227,10 +230,22 @@ export default class ProjectActivityAppContainer extends React.PureComponent {
     });
   };
 
-  firstLoadData(query /*: Query */) {
+  getTopLevelComponent = (component /*: Component */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)